kat/src/ecc384_kat.rs

```diff
@@ -44,6 +44,22 @@ const SIGNATURE: Ecc384Signature = Ecc384Signature {
     ]),
 };
 
+const KEY_GEN_PRIV_KEY: Array4x12 = Array4x12::new([
```

Comment:
If space is such a concern, couldn't we use these keys for the signature verification tests as well, and delete `SIGN_PRIV_KEY` and `SIGN_PUB_KEY`?

Reply:
Thanks for the feedback. I am actually making the change of combining these two tests into one. For this reason and because of the time this test takes in general.